latch_cli/snakemake/single_task_snakemake.py

```diff
@@ -291,5 +293,57 @@ def get_wildcards(self, requested_output, wildcards_dict=None):
 
 RRule.get_wildcards = get_wildcards
 
+
+def get_docker_cmd(
+    img_path,
+    cmd,
+    args="",
+    quiet=False,
+    envvars=None,
+    shell_executable=None,
+    container_workdir=None,
+    is_python_script=False,
+):
+    if shell_executable is None:
+        shell_executable = "sh"
+    else:
+        # Ensure to just use the name of the executable, not a path,
+        # because we cannot be sure where it is located in the container.
+        shell_executable = os.path.split(shell_executable)[-1]
+
+    docker_cmd = "docker run"
```

Comment:
use https://docs.python.org/3/library/shlex.html#shlex.join
to build command line strings